src/redux/Missions/missionsSlice.js

```diff
@@ -0,0 +1,74 @@
+import axios from "axios"; // eslint-disable-line
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const url = 'https://api.spacexdata.com/v3/missions';
+const initialState = {
+  missions: [],
+  status: 'idle',
+  error: null,
+};
+
+/* eslint-disable */
```

Comment:
I think it will be best to remove this `eslint-disable` since our microverse eslint check for state mutability automatically and It will easy for you to check for mutability as mention in #30 .